src/node_buffer.cc

```diff
@@ -1047,6 +1047,44 @@ static void EncodeUtf8String(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void EncodeInto(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  CHECK_GE(args.Length(), 3);
+  CHECK(args[0]->IsString());
+  CHECK(args[1]->IsUint8Array());
+  CHECK(args[2]->IsUint32Array());
+
+  Local<String> source = args[0].As<String>();
+  int source_utf8len = source->Utf8Length(isolate);
```

Comment:
Nothing important at all but could be directly a size_t here ?

Reply:
Thanks, I’ve removed it altogether because it shouldn’t be necessary anyway.